pypsa/plot.py

```diff
@@ -562,27 +559,37 @@ def iplot(n, fig=None, bus_colors='blue',
     fig : dict, default None
         If not None, figure is built upon this fig.
     bus_colors : dict/pandas.Series
-        Colors for the buses, defaults to "b"
-    bus_colorscale : string
-        Name of colorscale if bus_colors are floats, e.g. 'Jet', 'Viridis'
+        Colors for the buses, defaults to "cadetblue". If bus_sizes is a
+        pandas.Series with a Multiindex, bus_colors defaults to the
+        n.carriers['color'] column.
+    bus_alpha : float
+        Adds alpha channel to buses, defaults to 1.
+    bus_sizes : float/pandas.Series
+        Sizes of bus points, defaults to 10.
+    bus_cmap : plt.cm.ColorMap/str
+        If bus_colors are floats, this color map will assign the colors
     bus_colorbar : dict
         Plotly colorbar, e.g. {'title' : 'my colorbar'}
-    bus_sizes : dict/pandas.Series
-        Sizes of bus points, defaults to 10
-    bus_text : dict/pandas.Series
+    bus_text : pandas.Series
         Text for each bus, defaults to bus names
-    line_colors : dict/pandas.Series
-        Colors for the lines, defaults to "g" for Lines and "cyan" for
-        Links. Colors for branches other than Lines can be
-        specified using a pandas Series with a MultiIndex.
+    line_colors : str/pandas.Series
+        Colors for the lines, defaults to 'rosybrown'.
+    link_colors : str/pandas.Series
+        Colors for the links, defaults to 'darkseagreen'.
+    transfomer_colors : str/pandas.Series
+        Colors for the transfomer, defaults to 'orange'.
     line_widths : dict/pandas.Series
-        Widths of lines, defaults to 2. Widths for branches other
-        than Lines can be specified using a pandas Series with a
-        MultiIndex.
-    line_text : dict/pandas.Series
-        Text for lines, defaults to line names. Text for branches other
-        than Lines can be specified using a pandas Series with a
-        MultiIndex.
+        Widths of lines, defaults to 1.5
+    link_widths : dict/pandas.Series
+        Widths of links, defaults to 1.5
+    transformer_widths : dict/pandas.Series
+        Widths of transformer, defaults to 1.5
+    line_text : pandas.Series
+        Text for lines, defaults to line names.
+    link_text : pandas.Series
+        Text for links, defaults to line names.
+    tranformer_text : pandas.Series
+        Text for transformers, defaults to line names.
```

Comment:
```suggestion
    line_text : pandas.Series
        Text for lines, defaults to line names.
    link_text : pandas.Series
        Text for links, defaults to link names.
    tranformer_text : pandas.Series
        Text for transformers, defaults to transformer names.
```
This is what you meant right?